vex-riscv/bittide-cpu/src/main/scala/bittide/BittideCpu.scala

```diff
@@ -0,0 +1,85 @@
+// SPDX-FileCopyrightText: 2022 Google LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package bittide
+
+import spinal.core._
+import spinal.lib._
+import vexriscv.plugin._
+import vexriscv.{VexRiscv, VexRiscvConfig, plugin}
+
+object BittideCpu extends App{
+  def cpu() : VexRiscv = {
+    val config = VexRiscvConfig(
+      plugins = List(
+        new IBusSimplePlugin(
+          resetVector = 0x80000000l,
+          cmdForkOnSecondStage = false,
+          cmdForkPersistence = false,
+          prediction = NONE,
+          catchAccessFault = true,
+          compressedGen = true // C extension
```

Comment:
I will add the `FpuPlugin`, but that requires the use of the `DBusCachedPlugin` which might need a bit more testing, so I'll do it after I addressed the other review comments :+1:

Reply:
The `DBusCachedPlugin` doesn't really want to translate to HDL, but even then I saw that it uses the Wishbone Burst Type Identifier for burst requests. Our storages don't support that, so it might need a bit more work to add the F extension. I'll do this in a separate PR